### DISCUSSION TITLE
bind motion sensor to dimmer

### DISCUSSION THREAD

**Discussion by joukio** [Koenkk/zigbee2mqtt#8827] [Devices]
hi all,

got a question about motion sensors and such. I would like to bind a motion sensor to a dimmer/switch so when the sensor is triggered the light turns on. However, I don't seem to be able to bind the sensor to the dimmer I have (sensor = TS0202, dimmer = YSR-MINI-01_dimmer).
Now my question is, would this even be possible technically? I would like to do it via a binding rather then via home assistant, as I don't like the dependency and the delay.

regards
Jouk

**Comment by OmriMulti** [comment]
I’m trying to do the exact same thing!
I have an Aqara motion sensor and an IKEA bulb.
There has to be a way…

**Comment by Koenkk** [comment]
The only motion sensor I've seen capable of this is the IKEA motion sensor, all others sensors use a different cluster which lights don't understand.

**Reply by OmriMulti** [reply]
Thanks!
Going to buy one and test it!

But isn't there a way to modify the Aqara sensor?

**Reply by Koenkk** [reply]
It will require a different firmware (not sure if that is even possible)
